Dockerfile

```diff
@@ -4,11 +4,11 @@ FROM ekino/java7
 MAINTAINER fronton@ekino.com
 
 ENV WM_PACKAGE wiremock
-ENV WM_VERSION 1.55
+ENV WM_VERSION 2.1.11
```

Comment:
You should use `ARG` https://docs.docker.com/engine/reference/builder/#/arg to be able to easily change the version